Source/Scene/AttributeType.js

```diff
@@ -102,4 +103,72 @@ AttributeType.getMathType = function (attributeType) {
   }
 };
 
+/**
```

Comment:
@sanjeetsuhag question: is this AttributeType for internal use only? and is it only used for ModelExperimental?

If it's only internal ModelExperimental, consider naming it `ModelExperimentalAttributeType` and moving into the `Scene/ModelExperimental` directory. 

If it's only internal, consider `Renderer/` instead.